{"uid":"2f513362052a5c38","name":"STP: Complete Loan Journey for STP user","fullName":"SanityTest.LoanJourneyFlowForSTP.testSTPUserFullJourney","historyId":"7b63e61c69ea3c6bad3845aee9beb0c7","time":{"start":1751866074805,"stop":1751866084269,"duration":9464},"description":"Runs the full STP loan journey from login to application review, reusing modular methods.","descriptionHtml":"<p>Runs the full STP loan journey from login to application review, reusing modular methods.</p>\n","status":"broken","statusMessage":"Database update execution failed","statusTrace":"java.lang.RuntimeException: Database update execution failed\n\tat utils.DatabaseUtils.executeUpdate(DatabaseUtils.java:122)\n\tat utils.StepBypassUtils.makeUserEntryForSTP(StepBypassUtils.java:299)\n\tat TestUtils.ModularJourneySteps.makeUserEntryForSTP(ModularJourneySteps.java:441)\n\tat TestUtils.LoanJourneyFlow.completeSTPUserJourney(LoanJourneyFlow.java:1537)\n\tat SanityTest.LoanJourneyFlowForSTP.testSTPUserFullJourney(LoanJourneyFlowForSTP.java:21)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\n\tat org.testng.TestRunner.run(TestRunner.java:616)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\n\tat org.testng.TestNG.run(TestNG.java:1079)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\nCaused by: java.sql.SQLException: Column count doesn't match value count at row 1\n\tat com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:130)\n\tat com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)\n\tat com.mysql.cj.jdbc.StatementImpl.executeUpdateInternal(StatementImpl.java:1334)\n\tat com.mysql.cj.jdbc.StatementImpl.executeLargeUpdate(StatementImpl.java:2084)\n\tat com.mysql.cj.jdbc.StatementImpl.executeUpdate(StatementImpl.java:1245)\n\tat utils.DatabaseUtils.executeUpdate(DatabaseUtils.java:116)\n\t... 31 more\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"setUp","time":{"start":1751866071645,"stop":1751866074757,"duration":3112},"description":"","status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false,"stepsCount":0,"hasContent":false}],"testStage":{"description":"Runs the full STP loan journey from login to application review, reusing modular methods.","status":"broken","statusMessage":"Database update execution failed","statusTrace":"java.lang.RuntimeException: Database update execution failed\n\tat utils.DatabaseUtils.executeUpdate(DatabaseUtils.java:122)\n\tat utils.StepBypassUtils.makeUserEntryForSTP(StepBypassUtils.java:299)\n\tat TestUtils.ModularJourneySteps.makeUserEntryForSTP(ModularJourneySteps.java:441)\n\tat TestUtils.LoanJourneyFlow.completeSTPUserJourney(LoanJourneyFlow.java:1537)\n\tat SanityTest.LoanJourneyFlowForSTP.testSTPUserFullJourney(LoanJourneyFlowForSTP.java:21)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\n\tat org.testng.TestRunner.run(TestRunner.java:616)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\n\tat org.testng.TestNG.run(TestNG.java:1079)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\nCaused by: java.sql.SQLException: Column count doesn't match value count at row 1\n\tat com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:130)\n\tat com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)\n\tat com.mysql.cj.jdbc.StatementImpl.executeUpdateInternal(StatementImpl.java:1334)\n\tat com.mysql.cj.jdbc.StatementImpl.executeLargeUpdate(StatementImpl.java:2084)\n\tat com.mysql.cj.jdbc.StatementImpl.executeUpdate(StatementImpl.java:1245)\n\tat utils.DatabaseUtils.executeUpdate(DatabaseUtils.java:116)\n\t... 31 more\n","steps":[{"name":"Entering mobile number: 8509003772","time":{"start":1751866075437,"stop":1751866076528,"duration":1091},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"mobileNumber","value":"8509003772"}],"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false,"stepsCount":0,"hasContent":true}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":0,"attachmentStep":false,"stepsCount":1,"hasContent":true},"afterStages":[{"name":"tearDown","time":{"start":1751866084277,"stop":1751866084465,"duration":188},"description":"","status":"passed","steps":[],"attachments":[{"uid":"b0f9c2121f177eef","name":"All Failed APIs","source":"b0f9c2121f177eef.txt","type":"text/plain","size":607}],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":1,"attachmentStep":false,"stepsCount":0,"hasContent":true}],"labels":[{"name":"package","value":"SanityTest.LoanJourneyFlowForSTP"},{"name":"testClass","value":"SanityTest.LoanJourneyFlowForSTP"},{"name":"testMethod","value":"testSTPUserFullJourney"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"stashfin-website"},{"name":"subSuite","value":"SanityTest.LoanJourneyFlowForSTP"},{"name":"host","value":"MacBook-Pro.local"},{"name":"thread","value":"47172@MacBook-Pro.local.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":[]},"source":"2f513362052a5c38.json","parameterValues":[]}